src/components/StoryCreationComponent.js

```diff
@@ -12,45 +12,29 @@ import { Col, Row } from 'reactstrap';
 
 import '../styles/AttributeDeselector.css';
 
+//React quill
+import * as ReactQuill from 'react-quill'; // Typescript
```

Comment:
This import syntax is for typescript, you prolly wanna use this:
`import ReactQuill from 'react-quill';`

Reply:
Done.... Though, sometimes it's hard to tell between reactjs and typescript.